userguide/sharing.rst

```diff
@@ -53,10 +55,10 @@ These types of shares and services are available:
 
 * :ref:`SMB <Windows (SMB) Shares>`: Server Message Block shares, also
   known as Common Internet File System (CIFS) shares, are accessible
-  by Windows, Mac OS X, Linux, and BSD computers. Access is slower
+  by Windows, macOS, Linux, and BSD computers. Access is slower
   than an NFS share due to the single-threaded design of Samba. SMB
-  provides more configuration options than NFS and is a good choice
-  on a network for Windows systems. However, it is a poor choice if
+  provides more configuration options than NFS and is the best choice
```

Comment:
"best" is subjective.  I think the original "a good choice" is correct.  NFS is much more performant if the client software is available.

Reply:
Will revert my changes to this paragraph.